src/AssemblerAlign.cpp

```diff
@@ -302,7 +302,9 @@ void Assembler::computeAlignments(
     // Store alignmentInfos found by each thread in the global alignmentInfos.
     cout << timestamp << "Storing the alignment info objects." << endl;
     alignmentData.createNew(largeDataName("AlignmentData"), largeDataPageSize);
-    compressedAlignments.createNew(largeDataName("CompressedAlignments"), largeDataPageSize);
+    if (storeAlignments) {
```

Comment:
This change in unrelated to this work and will be made by @bagashe as part of a separate PR. Please do not include this change in this PR.